Command/CheckIpAddressIsBlacklistedCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace Azine\MailgunWebhooksBundle\Command;
+
+use Azine\MailgunWebhooksBundle\Services\AzineMailgunService;
+use Azine\MailgunWebhooksBundle\Services\HetrixtoolsService\HetrixtoolsServiceResponse;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Azine\MailgunWebhooksBundle\Entity\Repositories\MailgunEventRepository;
+use Doctrine\ORM\EntityManager;
+use Azine\MailgunWebhooksBundle\Services\HetrixtoolsService\AzineMailgunHetrixtoolsService;
+use Azine\MailgunWebhooksBundle\Services\AzineMailgunMailerService;
+
+/**
+ * Checks if the last ip address from MailgunEvent entity is in blacklist
+ *
+ */
+class CheckIpAddressIsBlacklistedCommand extends ContainerAwareCommand
```

Comment:
Command should be made Symfony4-ready. 
See https://github.com/azine/AzineMailgunWebhooksBundle/issues/21 and https://github.com/azine/AzineMailgunWebhooksBundle/pull/23/files